astro/src/content/docs/apis/_login-id-field.mdx

```diff
@@ -0,0 +1,22 @@
+import APIField from 'src/components/api/APIField.astro';
+import InlineField from 'src/components/InlineField.astro';
+import Aside from 'src/components/Aside.astro';
+
+<APIField name="loginId" type="String" required={!props.optional}>
+  The login identifier of the {props.user_label || 'user'}. The login identifier can be one of the following (see <InlineField>{props.login_id_type_field}</InlineField> for information on which of these identifiers will be used):
+  * Email address
```

Comment:
Intentionally veered away from using <InlineField>email when describing loginId to make it clear the actual user's info should go there, not the literal string email.